src/engraving/libmscore/hairpin.cpp

```diff
@@ -625,9 +625,9 @@ Sid Hairpin::getPropertyStyle(Pid pid) const
     case Pid::LINE_STYLE:
         return isLineType() ? Sid::hairpinLineLineStyle : Sid::hairpinLineStyle;
     case Pid::DASH_LINE_LEN:
-        return isLineType() ? Sid::hairpinLineDashLineLen : Sid::NOSTYLE;
+        return Sid::hairpinLineDashLineLen;
```

Comment:
can't say I understand why these shouldn't be used for any sort of hairpin - as it is, there doesn't appear to be away of changing these style values for the whole score anyway